packages/components/src/modal/style.scss

```diff
@@ -48,18 +40,18 @@
 
 @keyframes components-modal__appear-animation {
 	from {
-		margin-top: $grid-unit-40;
+		transform: translateY($grid-unit-40);
 	}
 	to {
-		margin-top: 0;
+		transform: translateY(0);
 	}
 }
 
 // Fix header to the top so it is always there to provide context to the modal
 // if the content needs to be scrolled (for example, on the keyboard shortcuts
 // modal screen).
 .components-modal__header {
-	box-sizing: border-box;
+	border-radius: $radius-block-ui $radius-block-ui 0 0;
```

Comment:
Does the parent border radius not affect the inner contents? Some overflow hidden stuff?

Reply:
Yes, it was added because I thought an overflow hidden on the frame element would interfere with sticky positioning within the content element but in this case it works fine 😌 . So I've gladly removed this (and the other added to the content element) in favor of the overflow hidden.